.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: go
 
+arch:
+  - amd64
+  - arm64
```

Comment:
This is most likely going to slow our PR workflow way down. Because travis takes a very long time (on the order of minutes - I've seen it take 8 minutes or more) to even start up a arm64 machine. 

We might want to comment those out until someone implements that trick to spawn tests that don't block tests from passing and allowing PRs to be merged.

Reply:
I'm not sure why, but on this PR travis actually was faster on ARM than on AMD: https://travis-ci.org/github/kiali/kiali/builds/677964501. Also, on the merge commit, it was faster: https://travis-ci.org/github/kiali/kiali/builds/680258920

Reply:
I know why. Because this kiali repo is using the legacy travis integration. For some reason using that goes fast. I saw this same behavior over in kiali-op repo. When I moved to the GitHub Apps integration, that's when it was slow.